### PR TITLE
fix: log video path if exists, regardless of compression

### LINKED ISSUE #25945: Location of recorded video is not printed if videoCompression: false
### Current behavior

If you're recording with videoCompression set to some number (other than 0), a section for Video output is displays to the terminal like shown below. This information contains progress around compression time and time it takes to process the video in general, but it also includes a link to the video which is valuable to see where the video is recorded.

Screen Shot 2023-02-24 at 2 35 30 PM

If you turn off videoCompression, setting it to false - this entire section is not printed. So you can't see the location of the recorded video. There was a recorded video below, but no one would know.

Screen Shot 2023-02-24 at 2 37 05 PM

### Desired behavior

This line of code, as well as the Video section title itself should still print when videoCompression is set to false. https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/print-run.ts#L540

This isn't happening because this entire section of printing in only ocurring if there is progression in processing the video. Really the link to the video should be separated out into a section that's always printed when video is present. https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/video_capture.ts#L371:L371

### Test code to reproduce

Run the code below in any project.

`cypress.config.js`
```js
const { defineConfig } = require('cypress')

module.exports = defineConfig({
  videoCompression: false
})
```

### Cypress Version

12.5.1

### Node version

16.16.1

### Operating System

Mac

### Debug Logs

_No response_

### Other

_No response_

### PR DESCRIPTION
**Pull request by AtofStryker**
- Closes #25945

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/server/lib/util/print-run.ts [hunk 1]
```diff
@@ -553,3 +541,21 @@ export function displayVideoCompressionProgress (opts: { videoName: string, vide
...
+
+export const printVideoHeader = () => {
+  console.log('')
+
+  terminal.header('Video', {
+    color: ['cyan'],
+  })
+}
+
+export const printVideoPath = (videoName?: string) => {
```
Comment (mjhenkes) [inline packages/server/lib/util/print-run.ts:553]: in what cases can video name be optional?

File: packages/server/lib/util/print-run.ts [hunk 2]
```diff
@@ -553,3 +541,21 @@ export function displayVideoCompressionProgress (opts: { videoName: string, vide
...
+  if (videoName !== undefined) {
+    console.log('')
```
Comment (mjhenkes) [inline packages/server/lib/util/print-run.ts:555]: are these just page breaks? could you condense this to a '\n' in the main console log? (mostly just curious)

**Review by mjhenkes — APPROVED** [review]

File: packages/server/lib/util/print-run.ts (hunk 1 above)
Comment (AtofStryker) [reply packages/server/lib/util/print-run.ts:553]: I am not sure but wanted to be safe here and make sure we don't print if its absent

File: packages/server/lib/util/print-run.ts (hunk 2 above)
Comment (AtofStryker) [reply packages/server/lib/util/print-run.ts:555]: pretty much just page breaks. We might be able to get these to `\n` but honestly the snapshots and console printing is so sensitive I am hesitant to touch it

**Review by ryanthemanuel — APPROVED** [review]

**Review by emilyrohrbough — COMMENTED** [review]
LGTM after updating to include a changelog 👍🏻

**Comment by ryanthemanuel** [comment]
Looks good, but let's add a changelog entry and switch this to a bug fix.

**Pull request merged**